### PR TITLE
Misc. improvements

### PR DESCRIPTION
**Pull request by erikarvstedt**
Some of these are required for the upcoming [mempool flake](https://github.com/erikarvstedt/nix-bitcoin-mempool).

### CODE REVIEW

File: helper/start-bash-session.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Start an interactive bash session in the current bash environment.
```
Comment (jonasnick) [inline helper/start-bash-session.sh:1]: Looks like this is currently unused. What is it for?

File: modules/netns-isolation.nix [hunk 2]
```diff
@@ -297,6 +299,7 @@ in {
         id = 31;
         connections = [ "bitcoind" ];
       };
+      # id = 32 reserved for mempool
```
Comment (jonasnick) [inline modules/netns-isolation.nix:302]: How about "mempool" -> "mempool extension flake" to clarify that mempool isn't part of this code base?

Wouldn't it be better to explicitly reserve a different range for extension flakes, e.g., anything > 1000?

File: helper/start-bash-session.sh (hunk 1 above)
Comment (erikarvstedt) [reply helper/start-bash-session.sh:1]: It's used in the mempool flake:
https://github.com/erikarvstedt/nix-bitcoin-mempool/blob/a47cc2d5a1c08b32eee79fa079bbadbc7bf5cf57/pkgs/mempool/generate.sh#L6

File: modules/netns-isolation.nix (hunk 2 above)
Comment (erikarvstedt) [reply modules/netns-isolation.nix:302]: Mempool will be added as a nix-bitcoin module when the backend bug is resolved.

New extension flakes will be assigned `id`s >= 100. (`id`s map to the last byte of a IPv4 address.)

Comment (erikarvstedt) [reply modules/netns-isolation.nix]: Done.

New extension flakes will be assigned `id`s >= 100. (`id`s map to the last byte of a IPv4 address.)

File: helper/start-bash-session.sh (hunk 1 above)
Comment (jonasnick) [reply helper/start-bash-session.sh:1]: Could you add a comment explaining how this is related to nix-bitcoin if it's unused in nix-bitcoin ("useful for flakes" or something)?

Comment (erikarvstedt) [reply helper/start-bash-session.sh:1]: Done.

**Review by jonasnick — APPROVED** [review]
ACK addfa8ec6bab320de0b808fd76809635947377e8

**Pull request merged**